aries.mk

```diff
@@ -67,9 +67,9 @@ PRODUCT_COPY_FILES += \
     device/sony/aries/rootdir/system/etc/tfa98xx/VoiceCallEarpice_top.preset:/system/etc/tfa98xx/VoiceCallEarpice_top.preset \
     device/sony/aries/rootdir/system/etc/tfa98xx/VoiceCallEarpice_top.eq:/system/etc/tfa98xx/VoiceCallEarpice_top.eq
 
-PRODUCT_NAME := aosp_d5803
+PRODUCT_NAME := aries
```

Comment:
Can't we have a separate aries.mk that would include aosp_d5803 and just rewrite what we need ? Would make it easier to track upstream changes.